tests/python/sdk/test_pytorch.py

```diff
@@ -43,8 +44,13 @@ def _common_setup(
         api_client.configuration.logger[k] = logger
 
 
-def _disable_api_requests(monkeypatch: pytest.MonkeyPatch) -> None:
-    def disabled_request(*args, **kwargs):
+def _disable_api_requests(monkeypatch: pytest.MonkeyPatch, exceptions: Container[str] = ()) -> None:
```

Comment:
Probably, the word exceptions is a bit misleading. I suggest changing it to `excludes` or `whitelist`.

Reply:
You're right. I changed the naming.